app/models/SubmissionsPages.scala

```diff
@@ -19,30 +19,43 @@ package models
 import play.api.mvc.QueryStringBindable
 
 case class SubmissionsPages(
-  rejectedPageNumber: Int = SubmissionsPages.DefaultPageNumber,
+  otherPageNumber: Int = SubmissionsPages.DefaultPageNumber,
```

Comment:
Nit: Can `other` be `submitted`. It would be less ambiguous when reading.

Reply:
I thought of this too. As I picked it up, _other_ was already in use, and as its calculated in *EnhancedStatus* to be a set comprised of all statuses _other than_ those _rejected_, _cancelled_ or _action needed_ I thought it best to leave it as is?